api/src/pcapi/core/offerers/models.py

```diff
@@ -784,6 +784,8 @@ class Offerer(
     offererProviders: list["OffererProvider"] = sa.orm.relationship("OffererProvider", back_populates="offerer")
     thumb_path_component = "offerers"
 
+    dmsToken: str = Column(Text, nullable=False, unique=True)
```

Comment:
La contrainte NOT NULL ne pourra être appliquée qu'après le passage du script. En attendant, la colonne doit être NOT NULLable : 

```suggestion
    dmsToken: str = Column(Text, nullable=True, unique=True)
```